packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -241,7 +241,7 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
             className={this._classNames.calloutMain}
             onDismiss={this.dismiss}
             onScroll={onScroll}
-            shouldRestoreFocus={true}
+            shouldRestoreFocus={this.props.shouldRestoreFocus !== undefined ? this.props.shouldRestoreFocus : true}
```

Comment:
I wonder if you can do `let {shouldRestorFocus = true } = this.props` and if that would  work correctly. I think destructing in that way does the null/undefined check